beam_nuggets/io/kafkaio.py

```diff
@@ -0,0 +1,138 @@
+from apache_beam import PTransform, ParDo, DoFn, Create, pvalue, Windowing
+from apache_beam.transforms.window import GlobalWindows
+
+from kafka import KafkaConsumer, KafkaProducer
+import json
+
+class KafkaConsume(PTransform):
+    """A :class:`~apache_beam.transforms.ptransform.PTransform` for reading from an Apache Kafka topic. This is a streaming
+    Transform that never returns.
+
+    It outputs a :class:`~apache_beam.pvalue.PCollection` of
+    ``key-values:s``, each object is a Kafka message in the form (msg-key, msg)
+
+    Examples:
+        Consuming from a Kafka Topic `notifications` ::
+
+            import apache_beam as beam
+            from apache_beam.options.pipeline_options import PipelineOptions
+            from beam_nuggets.io import kafkaio
+
+            kafka_topic = 'notifications'
+            kafka_config = {"topic": "kafka_topic",
+                            "servers": "localhost:9092",
```

Comment:
I think you meant `bootstrap_servers` instead of `servers`, since this is what `KafkaConsumer` expects.